packages/server/src/api/com/atproto/repo.ts

```diff
@@ -57,10 +57,15 @@ export default function (server: Server) {
       after,
     )
 
+    const lastRecord = records.at(-1)
+    const lastUri = lastRecord && new AdxUri(lastRecord?.uri)
+
     return {
       encoding: 'application/json',
       body: {
         records,
+        // Paginate with `after` by default, paginate with `before` when using `reverse`.
```

Comment:
Just a note that this is the opposite convention as many of the views since we default here to an ascending sort, but the views descending sort.

Reply:
I wonder if we should flip these semantics just to keep it consistent 🤔 

similar to most other queries, you'd normally want to ask for most recent objects not first-created objects

Reply:
Got it!  So the default is descending and `reverse` will flip it to ascending.